src/nvim/version.c

```diff
@@ -80,8 +80,8 @@ static int included_patches[] = {
   784,
   //783,
   //782,
-  //781,
-  //780,
+  781,
+  //780 NA
```

Comment:
I removed the `NA` marker for now. It's more important to get the patches in at the moment.